src/app/store/reducers/ReferenceData.reducers.spec.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  LoadPreparersSuccess, LoadTestStations,
+  LoadTestStationsSuccess
+} from '@app/store/actions/ReferenceData.actions';
+import { initialReferenceDataState } from '@app/store/state/ReferenceDataState.state';
+import { ReferenceDataReducers } from '@app/store/reducers/ReferenceData.reducers';
+import { Preparer } from '@app/models/preparer';
+import { TestStation } from '@app/models/test-station';
+
+describe('Reference Data Reducer', () => {
+  describe('[LoadPreparersSuccess]', () => {
+    test('should update preparers', () => {
```

Comment:
I think we have to stick with it()
Also a more meaningful description will help, like
should update the ReferenceDataReducers with the correct state when the given initial state and action is given

up to you for the naming of variables but it would be more intelligible if result was named after the state or reducer itself rather than result.